docs/notebooks/markov-models.ipynb

```diff
@@ -137,7 +137,7 @@
     "The symbol etymology, which usually gets swept under the rug in mathematically-oriented papers, are as follows: \n",
```

Comment:
<p>I think it should be <code>shape=(len(gaussian_ems),)</code>  instead of <code>shape=(len(poisson_ems),)</code>  in <code>hmm_states</code> .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ericmjl/bayesian-analysis-recipes/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='84'/>

Reply:
<p>Good catch.</p>